### PR TITLE
Assignment10

### PR DESCRIPTION
**Pull request by prajjwalkumarpanzade**
CRUD operations on Blog app 
Create User and Post Model
Create User and Post Controller
Create User and Post View
Perform CURD operation on User and Post

### CODE REVIEW

File: Assignment10/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+class PostsController < ApplicationController
+  before_action :set_post, only: [:show, :edit, :update, :destroy, :search]
+  def index
+    @posts = Post.all
+  end
+  def new
```
Comment (samu80) [inline Assignment10/controllers/posts_controller.rb:8]: Add line space in between two methods

File: Assignment10/controllers/posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    set_post
+    @post.destroy
+    redirect_to action: :index
+  end
+  private
+  def set_post
+    @post = Post.find(params[:id])
+  end
+  def post_params
+    params.require(:post).permit(:title,:body,:user_id)
```
Comment (samu80) [inline Assignment10/controllers/posts_controller.rb]: Add space after comma
`params.require(:post).permit(:title, :body, :user_id)`

File: Assignment10/controllers/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: [:show, :edit, :update, :destroy]
+  def index
```
Comment (samu80) [inline Assignment10/controllers/users_controller.rb:4]: Add line space

File: Assignment10/controllers/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    else
+      render :edit
+    end
+  end
+  def destroy
+    set_user
+    @user.destroy
+    redirect_to action: :index
+  end
+  private
```
Comment (samu80) [inline Assignment10/controllers/users_controller.rb:44]: Here also, add proper line spaces

File: Assignment10/controllers/users_controller.rb [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  def set_user
+    @user = User.find(params[:id])
```
Comment (samu80) [inline Assignment10/controllers/users_controller.rb:46]: What if user with param[:id] not found?

File: Assignment10/controllers/posts_controller.rb (hunk 1 above)
Comment (prajjwalkumarpanzade) [reply Assignment10/controllers/posts_controller.rb:8]: done

File: Assignment10/controllers/users_controller.rb (hunk 4 above)
Comment (prajjwalkumarpanzade) [reply Assignment10/controllers/users_controller.rb:44]: done

File: Assignment10/controllers/users_controller.rb (hunk 5 above)
Comment (prajjwalkumarpanzade) [reply Assignment10/controllers/users_controller.rb:46]: done validation

File: Assignment10/controllers/posts_controller.rb (hunk 2 above)
Comment (prajjwalkumarpanzade) [reply Assignment10/controllers/posts_controller.rb]: done

File: Assignment10/controllers/users_controller.rb (hunk 3 above)
Comment (prajjwalkumarpanzade) [reply Assignment10/controllers/users_controller.rb:4]: done

**Review by samu80 — APPROVED** [review]

**Pull request merged**